docs/getting-started.md

```diff
@@ -90,39 +90,46 @@ Explore our CLI to find more queries you can run.
 
 ### Submitting a transaction
 
+An account has an address which is determined by the key pair
```

Comment:
I think we can elaborate here and more clearly relate it to the goal of submitting a transaction. This would basically move the explanation you do further down here.
```suggestion
To submit a transaction you need an account on the block chain. An account is controlled by a cryptographic key pair. Key pairs are managed by `radicle-registry-cli` and stored on disk. An account is identified by a SS58 address which is derived from the key pair.
An account has an address which is determined by the key pair
```

Reply:
Sorry, I messed up with that line, didn't mean to include it.

I like your proposal, although I think it overloads the user with very technical information without a practical context to it, or not even, I think.

For example:

> To submit a transaction you need an account on the block chain. 

As a user, I would ask _why_? The only answer I can think of is so to pay for tx fees, therefore we could better be more transparent in that respect.

Reply:
> As a user, I would ask _why_? The only answer I can think of is so to pay for tx fees, therefore we could better be more transparent in that respect.

Good point. I think it would make sense to add this.

> I like your proposal, although I think it overloads the user with very technical information without a practical context to it, or not even, I think.

I don’t think this is information overload—on the contrary. All the terms introduced and put into relation here are used further down without context. For example the sentence “You can always obtain the account addresses for all your key pair” is the first to mention an account address but fails to explain its relation to the a key pair.

In addition the sentence here explains why we need to generate a key pair. Something that is hinted at in the sentence “You need a key pair to author transactions” below.

Reply:
> I don’t think this is information overload

I wasn't clear: the information is good, I just missed the explanation as to why some bits were so; under that light I found the info to be overloading.

Anyway, I have reworked this part to integrate your latest suggestion, which I have slightly touched. Let me know how you like it now.